docs/astro/sql/operators/load_file.rst

```diff
@@ -0,0 +1,162 @@
+======================================
+load_file operator
+======================================
+
+When to Use load_file operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two use cases of the load_file operator.
+
+#. When we want a file to be loaded into a database table
+#. When we want a file to be converted to dataframe
+
+Case 1: Load into a database table
+    you need to pass the `output_table` param to the load_file operator to convert it to a table and the operator returns an instance of the table object passed in `output_table`. The output table will be created if it doesn't exist and will be replaced if it does. We can change this behavior with the `if_exists` parameter.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_1_start]
+       :end-before: [load_file_example_1_end]
+
+Case 2: Load into pandas dataframe
+    If you don't pass the `output_table` to the load_file operator it converts the file into a pandas dataframe and returns the reference to dataframe.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_2_start]
+       :end-before: [load_file_example_2_end]
+
+Parameters to use when loading a file to the database table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. **ndjson_normalize_sep** - This parameter is useful when the input file type is NDJSON. Since NDJSON file can be multidimensional, we normalize the data to two-dimensional data, so that it is suitable to be loaded into a table and this parameter is used as a delimiter for combining columns names if required.
```

Comment:
Priority-wise, it feels this should be the last option to be listed - since it is NDJSON specific.
We probably want to have `if_exists` and `output_table` at the top of the arguments list.

Reply:
Done.